client/JS/giphyDom.js

```diff
@@ -0,0 +1,23 @@
+const container = document.querySelector(".main");
+const createCards = (data) => {
+  container.textContent = "";
+  data.data.forEach((element) => {
+    let card = document.createElement("div");
+    card.classList.add("giphy");
+
+    let img = document.createElement("img");
+    img.classList.add("img-giphy");
+    img.setAttribute("src", `${element.images.downsized.url}`);
+    card.appendChild(img);
+
+    container.appendChild(card);
+  });
+};
+console.log(window.location);
+const searchParams = new URLSearchParams(window.location.search);
+const anime = searchParams.get("anime");
```

Comment:
what if the anime value is undefined what will happen :grimacing:

Reply:
If undefined, it will show random anime gifs.